cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+      schema:
+        $ref: '#/components/schemas/CardId'
+      description: Identifier (technical) for a card for which information is requested.
+
+    path_tokenId:
+      in: path
+      name: tokenId
+      required: true
+      schema:
+        $ref: '#/components/schemas/TokenId'
+      description: Identifier (technical) for a token for which information is requested.
+
+    path_transactionId:
+      in: path
+      name: transactionId
+      required: true
+      schema:
+        $ref: '#/components/schemas/TransactionId'
+      description: Identifier (technical) for a transaction for which information is requested.
+
+    limit:
+      in: query
+      name: limit
+      required: false
+      schema:
+        type: integer
+        format: int32
+        minimum: 1
+        examples:
+          - 25
+      description: Optional parameter for pagination. Maximum number of entries to be returned.
+
+    offset:
+      in: query
+      name: offset
+      required: false
+      schema:
+        type: integer
+        format: int32
+        minimum: 0
+        examples:
+          - 25
+      description: Optional parameter for pagination. The number of items to skip before starting to collect the result set.
+
+    client_in_header:
+      in: header
+      name: X-CorAPI-Client-ID
+      required: true
+      schema:
+        type: string
+      description: ID of the client forwarded to the provider.
+
+    correlation_in_header:
+      in: header
+      name: X-Correlation-ID
+      required: true
+      schema:
+        $ref: '#/components/schemas/Correlation'
+      description: Unique ID (defined by the caller) which will be reflected back in the response.
+
+    agent_in_header:
+      in: header
+      name: User-Agent
+      required: true
+      schema:
+        type: string
+      description: Name and version of the of the Client software.
+
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-API-Key
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: https://example.com/oauth/authorize
+          tokenUrl: https://example.com/oauth/token
+          scopes:
+            read: Grants read access
+            write: Grants write access
+
+  responses:
+    standard400:
+      headers:
+        Content-Type:
+          $ref: '#/components/headers/Content-Type'
+        Content-Language:
+          $ref: '#/components/headers/Content-Language'
+        X-Correlation-ID:
+          $ref: '#/components/headers/X-Correlation-ID'
+      description: >
+        Client Error - The request could not be processed due to a problem in the data provided.
+        This could be for any of the reasons of the HTML standard that would result in a 4xx code.
+      content:
+        application/problem+json:
+          schema:
+            $ref: '#/components/schemas/CommonErrorResponse'
+
+    standard500:
+      headers:
+        Content-Type:
```

Comment:
`Content-Type` MUST NOT be specified, it is implicitly given in the OpenAPI.

Reply:
Will be removed here and for standard400.